pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseDatawithEstimations.py

```diff
@@ -230,8 +230,15 @@ def write_weekly_deaths_xlsx_data(
 
     def test_get_case_data_with_estimations(self):
 
-        [read_data, make_plot, file_format, out_folder, no_raw] \
-            = [True, False, "json", self.path, False]
+        read_data = True
+        make_plot = False
+        file_format= 'json'
+        out_folder = self.path
+        no_raw = False
+        impute_dates = False
+        moving_average = 0
+        split_berlin = False
+        rep_date = False
```

Comment:
Since those inputs are different for every test they cant be defined in the beginning. Maybe it would be better if all defaults are defined in the beginning and only the non-defaults are defined in the test itself. But then it should be corrected for all unittests. Maybe we should open a Issue for that?